### PR TITLE
fix(QTM-305): Removes focus on first element

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-305

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review
- [ ] TypeScript updated

### A11y review
- [ ] A11y checks
  - [ ] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile

### CODE REVIEW

File: components/Modal/Modal.unit.test.jsx [hunk 1]
```diff
@@ -92,12 +92,34 @@ describe('<Modal />', () => {
...
+            <button type="button">Cancel</button>
+            <button type="button">Ok</button>
+          </Modal.Footer>
+        </Modal>,
+      );
+
+      const focusedElement = document.activeElement;
+
+      expect(focusedElement.nodeName).not.toBe('A');
+      expect(focusedElement.nodeName).toBe('BUTTON');
```
Comment (matheusrpp) [inline components/Modal/Modal.unit.test.jsx:122]: Question. Why two expects? Wouldn't just one already solve the job?

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

File: components/Modal/Modal.unit.test.jsx (hunk 1 above)
Comment (toncabral88) [reply components/Modal/Modal.unit.test.jsx]: I think it's necessary to evidence first item in focus can't be a link, but I notice description of test is wrong.

**Review by rapahaeru — APPROVED** [review]

**Review by matheusrpp — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

**Pull request merged**